homeassistant/components/growatt/sensor.py

```diff
@@ -0,0 +1,178 @@
+"""
+Support for Growatt Plant energy production sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.growatt/
+"""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_USERNAME, CONF_PASSWORD
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ["growatt==0.0.2"]
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+    }
+)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Growatt Plant sensor."""
+    import growatt
+
+    username = config.get(CONF_USERNAME)
```

Comment:
```suggestion
    username = config[CONF_USERNAME]
```

Because this is required field.